content/en/storage-providers/seal-workers/post-workers.md

```diff
@@ -0,0 +1,145 @@
+
+### Minminal spec requirements
+
+These are the minimal requirements for running each of the PoSt tasks:
+
+| **Task**    | **GPU**            | **VRAM** | **RAM** |
+|:-----------:|:------------------:|:--------:|:-------:|
+| winningPoSt | Highly recommended |          |         |
+| windowPoSt  | Highly recommended | 8\.5GiB  | 128GiB  |
+
+{{< alert icon="callout" >}}
+Although both tasks can be run with a powerful CPU, it is highly recommended to run on GPU as it speeds up the process significantly, and therefore reducing the chance to miss windowPoSt or winningPoSt
+{{< /alert >}}
+
+### Remote storage access
+
+The windowPoSt process requires reading random leafs of all the sealed sectors in a proving deadline, so when setting up windowPoSt workers one needs to consider how the workers can access those files. By default 
+
+At least one PoSt worker needs local read access to the sealed sectors. If you have multiple PoSt workers, workers without local sector access will ask workers with sector access to read challenges from their storage.
+
+The lotus-miner instance disables PoSt-work when a PoSt worker is connected, meaning that a single windowPoSt worker can not rely on reading challenges from the lotus-miner instance. And therefore needs read access to the sealed sectors. 
+
+{{< alert icon="warning" >}}
+Storage providers should design their sealed sector access depending on their setup size and also redundancy required. If only one PoSt worker has access to the sealed sector that can create a single point of failure. If you have multiple partitions in a deadline, having multiple PoSt workers reading challenges from the same source may cause unwanted I/O load.
+{{< /alert >}}
+
+### Environment variables
+
+Ensure that workers have access to the following environment variables when they run. These are similar to those used by the Miner daemon ([explained in the setup guide]({{< relref "setup" >}})):
+
+```
+# MINER_API_INFO as obtained before
+export MINER_API_INFO:<TOKEN>:/ip4/<miner_api_address>/tcp/<port>/http`
+export MARKETS_API_INFO:<TOKEN>:/ip4/<miner_api_address>/tcp/<port>/http`
+export BELLMAN_CPU_UTILIZATION=0.875      # optimal value depends on exact hardware
+export FIL_PROOFS_MAXIMIZE_CACHING=1
+export FIL_PROOFS_USE_GPU_COLUMN_BUILDER=1 # when GPU is available
+export FIL_PROOFS_USE_GPU_TREE_BUILDER=1   # when GPU is available
+export FIL_PROOFS_PARAMETER_CACHE=/fast/disk/folder # > 100GiB!
+export FIL_PROOFS_PARENT_CACHE=/fast/disk/folder2   # > 50GiB!
+```
+
+The PoSt workers will fail to start if the file descriptor limit is not set high enough. Raise the the file descriptor limit temporarily before running with `ulimit -n 1048576` or permanently by following the [Permanently Setting Your ULIMIT System Value](https://github.com/filecoin-project/lotus/discussions/6198) guide.
+
```

Comment:
Can we add a suggestion to copy over the proofs from miner here as an alternative? It would be quicker and we all do it anyways.

Reply:
Good idea, I will add that as section between the env-vars and the FD-descriptor limit section